src/opcodes.py

```diff
@@ -0,0 +1,47 @@
+class Opcode:
+    '''This class is essentially a type to ensure that a piece of data 
+    when created is a valid operation code, 
+    or at least conforms to the format of a signed four digit base ten number'''
+
+    def __init__(self, raw):
+        self.raw = raw
+        self.sign = ""
+        self.op_list = {10:"READ", 11:"WRITE", 20:"LOAD", 21:"STORE", 30:"ADD", 31:"SUBTRACT", 32:"DIVIDE", 33:"MULTIPLY", 40:"BRANCH", 41:"BRANCHNEG", 42:"BRANCHZERO", 43:"HALT"}
```

Comment:
Consider making this a private property by naming it `self.__op_list`. It could also be a `@staticmethod` but don't worry about that if you've never implemented that sort of thing before.